### PR TITLE
use Collections.emptyList() #1095

### PR DESCRIPTION
**Pull request by anirudhbagri**
Collections.emptyList() instead of Collections.EMPTY_LIST
#1095
## Type of change

 - [ ] Bug fix (non-breaking change which fixes an issue)
 - [ ] Feature (non-breaking change which adds functionality)
 - [ ] Breaking change (fix or feature that would cause existing functionality to change
 - [ ] Chore (non-breaking change which doesn't affect codebase;
   test, version modification, documentation, etc.)

## Checklist
 - [ ] I have read the [contributing guidelines](https://www.eclipse.org/jkube/contributing)
 - [ ] I signed-off my commit with a user that has signed the [Eclipse Contributor Agreement](https://www.eclipse.org/legal/ECA.php)
 - [ ] I Added [CHANGELOG](../CHANGELOG.md) entry
 - [ ] I have implemented unit tests to cover my changes
 - [ ] I have updated the [documentation](../kubernetes-maven-plugin/doc) accordingly
 - [ ] No new bugs, code smells, etc. in [SonarCloud](https://sonarcloud.io/dashboard?id=jkubeio_jkube) report
 - [ ] I tested my code in Kubernetes
 - [ ] I tested my code in OpenShift

### CODE REVIEW

**Comment by anirudhbagri** [comment]
Hi @manusa,

Can you assign this to me?

**Comment by manusa** [comment]
> Can you assign this to me?

Yes, but you need to comment on the issue itself so that GH allows me to assign to you.

**Comment by anirudhbagri** [comment]
done.

File: jkube-kit/jkube-kit-wildfly-jar/src/test/java/org/eclipse/jkube/wildfly/jar/generator/WildflyJARGeneratorTest.java [hunk 1]
```diff
@@ -57,7 +57,7 @@
     @Test
     public void notApplicable() throws IOException {
         WildflyJARGenerator generator = new WildflyJARGenerator(createGeneratorContext());
-        assertFalse(generator.isApplicable((List<ImageConfiguration>) Collections.EMPTY_LIST));
+        assertFalse(generator.isApplicable((List<ImageConfiguration>) Collections.emptyList()));
```
Comment (rohanKanojia) [inline jkube-kit/jkube-kit-wildfly-jar/src/test/java/org/eclipse/jkube/wildfly/jar/generator/WildflyJARGeneratorTest.java:60]:
```suggestion
        assertFalse(generator.isApplicable(Collections.emptyList()));
```

**Comment by rohanKanojia** [comment]
ECA check seems to be failing again :-(

Did you configure git username and email (needs to be same as email which you used for Eclipse Account) before committing changes?
```
# (Optional): Configure Git User and Email
git config user.name "Your Name"
git config user.email "your_email@example.com"
```

If not, you can amend your commit to change author:
```
git commit --amend --author="Your Name<your_email@example.com>"
```

**Comment by rohanKanojia** [comment]
polite ping, any update on this. You just need to update your commit with Signed Off Footer and we're good to go.

**Comment by anirudhbagri** [comment]
Hi @rohanKanojia.. I have squashed the commits and added Sign-Off, Can you take a look.

**Comment by rohanKanojia** [comment]
commit should be in `Signed-Off By` footer instead of `Authored by`

**Comment by anirudhbagri** [comment]
Okay.

**Comment by rohanKanojia** [comment]
`git commit --amend --signoff`  is what you need to do

**Pull request closed without merging**